src/lang/en-US.ts

```diff
@@ -155,6 +157,7 @@ const CTAs: ResolvedIntlConfig['messages'] = {
     'cta.back': `Back`,
     'cta.revoke': `Revoke`,
     'cta.goToDetails': `See Details`,
+    'cta.goToEntityPage': `See all`,
```

Comment:
I think making this key name more general might be good as I could see us using `See all` in other places.